scripts/linters/python_linter.py

```diff
@@ -417,34 +377,39 @@ def _lint_py_files(self, config_pylint, config_pycodestyle):
                 python_utils.PRINT('Linting Python files %s to %s...' % (
                     current_batch_start_index + 1, current_batch_end_index))
 
-            with linter_utils.redirect_stdout(stdout):
+            with linter_utils.redirect_stdout(sys.stdout):
                 # This line invokes Pylint and prints its output
                 # to the target stdout.
                 pylint_report = StringMessageStream()
                 pylinter = lint.Run(
                     current_files_to_lint + [config_pylint],
                     reporter=text.TextReporter(pylint_report),
                     exit=False).linter
+
+            with linter_utils.redirect_stdout(stdout):
                 # These lines invoke Pycodestyle and print its output
                 # to the target stdout.
                 style_guide = pycodestyle.StyleGuide(
                     config_file=config_pycodestyle)
                 pycodestyle_report = style_guide.check_files(
                     paths=current_files_to_lint)
 
-            if pylinter.msg_status != 0 or pycodestyle_report.get_count() != 0:
+            if pycodestyle_report.get_count() != 0:
                 summary_message = stdout.getvalue()
+                summary_messages.append(summary_message)
+                errors_found = True
+
+            if pylinter.msg_status != 0:
                 for message in pylint_report.read():
                     python_utils.PRINT(message)
                 pylint_error_messages = (
                     self._get_trimmed_error_output(pylint_report.read()))
                 summary_messages.append(pylint_error_messages)
-                python_utils.PRINT(summary_message)
```

Comment:
is this an expected change? (Deletion of print)

Reply:
Yes we don't need it now